libraries/plugins/elasticsearch/elasticsearch_plugin.cpp

```diff
@@ -316,7 +315,7 @@ void elasticsearch_plugin_impl::sendBulk(std::string _elasticsearch_node_url, bo
 elasticsearch_plugin::elasticsearch_plugin() :
    my( new detail::elasticsearch_plugin_impl(*this) )
 {
-   curl = curl_easy_init();
+   my->curl = curl_easy_init();
```

Comment:
I think it's better to move it to constructor function of `elasticsearch_plugin_impl`.